### PR TITLE
fix: handle iframe close as payment dismiss

### PR DESCRIPTION
**Pull request by troggy**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

- make closeHandle in client to resolve the promise of `requestPayment` call if any. This way closing the iframe with close handle is equal to pressing "Dismiss" button on payment screen

### CODE REVIEW

File: packages/client/src/index.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ type WhiteRabbitMessage = {
   }
 };
 
+type Response = {
```
Comment (mwaa) [inline packages/client/src/index.ts:59]: Why not reuse WhiteRabbitMessage? 
Either reuse or make changes to client to be aware of this new response

Comment (troggy) [reply packages/client/src/index.ts:59]: because these responses are different? 🤔

Comment (troggy) [reply packages/client/src/index.ts:59]: you are right, now they contain the same infromation so we can use the same type indeed. And maybe naming is not the best here.

But technically those could be different. In this chain of communication `streaming site ← client ← wallet`:
`WhiteRabbitMessage` — is the response WR client gets from the WR app/wallet (via messaging)
`Reponse` — is the response streaming site get's from WR client

Comment (troggy) [reply packages/client/src/index.ts]: adjusted. thanks!

**Comment by mwaa** [comment]
Looks Good

1. Reviewed Code
2. Tested functionality on local

**Pull request merged**